sdk/identity/azure-identity/tests/managed-identity-live/service-fabric/service_fabric.md

```diff
@@ -0,0 +1,190 @@
+
+- This sample requires access to an Azure subscription and required privileges to create resources.
+- [Powershell and the Az library are needed to run the deployments in the sample.](https://docs.microsoft.com/powershell/azure/install-az-ps)
+- [Azure CLI is used to deploy some resources.](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+- Docker is needed to build and push the sample containerized service. Docker should be using Linux containers for building the application images that are provided.
+
+### Clone this repository
+
+From a command prompt window, run
+```
+git clone https://github.com/Azure/azure-sdk-for-python --single-branch --branch master --depth 1
+cd azure-sdk-for-python/sdk/identity/azure-identity/tests/managed-identity-live/service-fabric
+```
+
+### Sections
+- [Resource setup](#set-up-resources)
+- [Application deployment](#set-up-and-deploy-the-applications)
+- [Test validation](#run-the-tests)
+
+## Set Up Resources
+
+You can skip to [Set Up and Deploy the Applications](#set-up-and-deploy-the-applications) if you have an existing Service Fabric cluster, key vault, storage account, container registry, and managed identity named "AdminUser".
+
+### Create a resource group
+
+From a command prompt window, run
+```
+az login
+az group create -n $RESOURCE_GROUP --location $LOCATION --subscription $SUBSCRIPTION_NAME
+```
+
+### Create a user-assigned managed identity
+
+From your command prompt window, run
+```
+az identity create -g $RESOURCE_GROUP -n AdminUser
+```
+
+Make note of the identity's principal ID, to use in the next step. Also make note of its client ID for a later step.
+
+### Create a key vault, certificate, and secret
+
+Create your key vault:
+```
+az keyvault create -g $RESOURCE_GROUP -n $KEY_VAULT_NAME --sku standard
+```
+
+After creating the vault, create a self-signed certificate in it using the [Azure Portal](https://azure.portal.com/). You'll need to insert some of this certificate's properties into the cluster template later on.
+
+Next, create a secret in the key vault. It can have any name (e.g. "TestSecret") and any value (e.g. "TestValue"). This secret will just be accessed by the Service Fabric applications to verify that they can access a resource and read contents using their identities.
+
+Finally, go to the "Access policies" blade for your vault. Under "Enable Access to:", check the boxes for "Azure Virtual Machines for deployment" and "Azure Resource Manager for template deployment". Be sure to click "Save" at the top of the page. These policies are necessary for your cluster deployment and applications' functionality.
+
+### Create an Azure Container Registry
+
+From your command prompt window, run
+```
+az acr create -g $RESOURCE_GROUP -n $ACR_NAME --admin-enabled --sku basic
```

Comment:
What about the location?

Reply:
From when I ran these commands myself, this seems to be similar to the vault creation. It automatically chose the same location as the specified resource group